usefulUtilities/hiFiCalendar/entities/calendarEntities.json

```diff
@@ -0,0 +1,463 @@
+{
```

Comment:
looks like this file applies to JUST HQ and even contains some private meeting data specific to when you exported the JSON. can you please make this JSON generic and free of private information so that the only things it rezzes are:
1. the occupant list (what is "Calendar_Atlantis_Main_RoomColorOccupant" ? is it needed?)
2. the "in use" etc text
3. the clock entity
4. the room schedule
5. the zone around the room

can you then please make a separate JSON that just rezzes the token server?

Reply:
Done